pkg/distro/fedora/images.go

```diff
@@ -508,6 +508,9 @@ func imageInstallerImage(workload workload.Workload,
 		img.AdditionalDrivers = append(img.AdditionalDrivers, installerConfig.AdditionalDrivers...)
 	}
 
+	// On Fedora anaconda needs dbus-broker, but isn't added when dracut runs.
```

Comment:
Is there a reason for this; are we missing a package in the set? Probably not since enabling the module doesn't fail but it *feels* weird :)

Reply:
Understood, let's leave it as is then :)